src/openrct2/ride/Vehicle.h

```diff
@@ -425,6 +425,12 @@ enum class MiniGolfAnimation : uint8_t
     Putt,
 };
 
+enum BoatHireSubState : uint8_t
+{
+    Normal,
+    EnteringReturnPosition
```

Comment:
Please add trailing commas to the end of enums and arrays, so that future additions to them only make one line change, and to have clang-format properly format them.

```suggestion
    EnteringReturnPosition,
```